src/main/java/top/yatt/dfargx/RegexComparator.java

```diff
@@ -0,0 +1,26 @@
+package top.yatt.dfargx;
+
+import top.yatt.dfargx.automata.DFA;
+import top.yatt.dfargx.automata.NFA;
+import top.yatt.dfargx.tree.SyntaxTree;
+
+public class RegexComparator {
+
+    /**
+     * Checks if the matching space of <code>regexp2</code> is contained by the matching space of <code>regexp1</code>.
+     */
+    public static final boolean isContained(String regexp1, String regexp2) {
```

Comment:
Better to call "contains" or find regexp1 from regexp2.

Reply:
Do you mean the method name? Sure, I can change it to whatever you think is better.

Reply:
Yes, I mean the method name. 

IMO, "isContained" always implies that left is contained by right, and "contains" indicates the opposite meaning.

e.g
[org.apache.commons.lang3.StringUtils.contains](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html)
[org.apache.commons.collections4.CollectionUtils.containsAll](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html)

So I think either changing the method name or just swapping the parameter order are better. What do you think?